centreon/src/Core/Macro/Application/Repository/ReadHostMacroRepositoryInterface.php

```diff
@@ -48,4 +48,13 @@ public function findByHostIds(array $hostIds): array;
      * @return Macro[]
      */
     public function findByHostId(int $hostId): array;
+
```

Comment:
Add in comment of the interface, that the host and host template are handled by this interface